game/hud/src/services/session/layout.ts

```diff
@@ -147,11 +142,11 @@ const initialState = () => {
     reset: FORCE_RESET_CODE,
     locked: true,
     widgets: {
-      Chat:{x:{anchor:-1,px:0},y:{anchor:1,px:302},size:{width:480,height:240},scale:1},
-      PlayerHealth:{x:{anchor:0,px:-294},y:{anchor:1,px:315},size:{width:350,height:200},scale:0.6},
-      EnemyTargetHealth:{x:{anchor:0,px:-18},y:{anchor:1,px:315},size:{width:350,height:200},scale:0.6},
-      FriendlyTargetHealth:{x:{anchor:0,px:-18},y:{anchor:1,px:195},size:{width:350,height:200},scale:0.6},
-      Respawn:{x:{anchor:0,px:-100},y:{anchor:0,px:-100},size:{width:200,height:200},scale:1},
+      Chat:{x:{anchor:-1,px:0},y:{anchor:1,px:302},locked:false,size:{width:480,height:240},scale:1},
+      PlayerHealth:{x:{anchor:0,px:-294},y:{anchor:1,px:315},locked:false,size:{width:350,height:200},scale:0.6},
+      EnemyTargetHealth:{x:{anchor:0,px:-18},y:{anchor:1,px:315},locked:false,size:{width:350,height:200},scale:0.6},
+      FriendlyTargetHealth:{x:{anchor:0,px:-18},y:{anchor:1,px:195},locked:false,size:{width:350,height:200},scale:0.6},
+      Respawn:{x:{anchor:0,px:-100},y:{anchor:0,px:-100},locked:false,size:{width:200,height:200},scale:1},
```

Comment:
not directly related to your code change, but I think we should come up with a way for this code to be contained in each widget. We shouldn't have to edit a generic file (layouts.ts) each time we add/change a widget